albs_github/graphql/client.py

```diff
@@ -90,6 +92,33 @@ def project_id(self) -> Optional[str]:
     def repository_id(self) -> Optional[str]:
         return self.__default_repository_id
 
+    @staticmethod
+    async def generate_token_for_gh_app(
+        path_to_gh_app_pem: str,
+        gh_app_id: str,
+        installation_id: str,
+    ) -> str:
+        with open(path_to_gh_app_pem, 'rb') as pem_file:
+            signing_key = jwk_from_pem(pem_file.read())
+        payload = {
+            'iat': int(time.time()),
+            'exp': int(time.time()) + 600,
+            'iss': gh_app_id,
+        }
+        jwt_instance = JWT()
+        encoded_jwt = jwt_instance.encode(payload, signing_key, alg='RS256')
+        access_url = f'https://api.github.com/app/installations/{installation_id}/access_tokens'
+        async with aiohttp.request(
+            'POST',
+            access_url,
+            headers={
+                'Authorization': f'Bearer {encoded_jwt}',
+                'Accept': 'application/vnd.github.v3+json',
+            },
```

Comment:
please, add `raise_for_status=True` and check on the web server side that we get token